fal/models/team.py

```diff
@@ -18,3 +21,27 @@ class Team(Base):
     season = relationship("Season", back_populates="teams")
     wildcards = relationship("WildcardUsage", back_populates="team")
     team_weekly_anime = relationship("TeamWeeklyAnime", back_populates="team")
+
+    @staticmethod
+    def get_team_from_database(name: str, season: Season, session: Session) -> Team:
+        """ Adds new team row to database if necessary, then return the team object"""
+        query = session.query(Team).filter(
+            Team.name == name,
+            Team.season_id == season.id
+        )
+        team = query.one_or_none()
+
+        if not team:
+            team = Team(name=name, season_id=season.id)
+            session.add(team)
+            session.commit()
+            query = session.query(Team).filter(
```

Comment:
Why query instead of returning the team object we just made? Is this just as a safeguard?

Reply:
You need the ID provided by mysql's auto-increment

Reply:
it's possible you don't actually need to query it and sqlalchemy might actually update the id field automatically. if so it would make testing a bit easier. I'll investigate that later

Reply:
I made the other two changes so you can approve this PR and unblock your PRs. I will tackle this one in a separate change